models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -15,7 +15,7 @@ SELECT
 FROM
 (
     VALUES
-
+    ("eth-ethereum","ethereum","ETH","{{ var('ETH_ERC20_ADDRESS') }}",18),
```

Comment:
this already exists in the `prices_native_tokens` spell

Reply:
or if it doesn't should likely?

Reply:
Yeah those aren't in the format we need here as they don't have a `chain` and `contract_address` value.

The goal is that we can just use `var(ETH_ERC20_ADDRESS)` in models and don't need any other custom logic to handle the native token case for prices or erc20 matching..

But I agree that having
1. a native ETH row in `prices_ethereum_tokens` with our own `ETH_ERC20_ADDRESS`
2. a native ETH row in `prices_native_tokens` with no address
3. a native ETH row in `tokens_native` with the WETH symbol and address.

Is not what we'd want, I'll try to think of a better setup that works for everything. Will need to look into the current uses of 2) and 3) first.

Reply:
https://dune.height.app/T-24057?task=T-27753